packages/manager/apps/okms/src/modules/secret-manager/pages/drawers/create-version-drawer/CreateVersionDrawerForm.component.tsx

```diff
@@ -66,6 +69,12 @@ export const CreateVersionDrawerForm = ({
   } = form;
 
   const handleSubmitForm = async (data: FormSchema) => {
+    trackClick({
+      location: PageLocation.funnel,
+      buttonType: ButtonType.link,
+      actionType: 'navigation',
+      actions: ['create_version', 'confirm'],
```

Comment:
I would change this for : 

```suggestion
      location: PageLocation.popup,
      buttonType: ButtonType.button,
      actionType: 'action',
      actions: ['create_version', 'confirm'],
```

Reply:
updated